src/server/ocpp/utils/OCPPUtils.ts

```diff
@@ -613,6 +614,10 @@ export default class OCPPUtils {
         userID: transaction.userID,
         endedAt: Utils.convertToDate(meterValue.timestamp),
       } as Consumption;
+      // Handle current Connector limitation
+      await OCPPUtils.addConnectorLimitationToConsumption(tenantID, chargingStation, transaction.connectorId, consumption);
+      // Handle current Site Area limitation
+      await Utils.addSiteLimitationToConsumption(tenantID, chargingStation.siteArea, consumption);
```

Comment:
It time consuming to have these method here, move it just after the 'if' below if you agree:
```
      // Handle Consumption (Wh/kWh)
      } else if (OCPPUtils.isEnergyActiveImportMeterValue(meterValue)) {
```

Reply:
You are right. I moved it.
The issue will be fully solved, when the currentConnectorLimit Method is working correctly.